From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Patrick McCarty <patrick.mccarty@intel.com>
Date: Wed, 16 Dec 2020 07:35:33 +0000
Subject: [PATCH] Fix build with autoconf 2.70

- `autoreconf` from autoconf 2.70 requires IT_PROG_INTLTOOL instead of
  AC_PROG_INTLTOOL to enable the invocation of `intltoolize`, probably
  because the latter macro is deprecated.

- The IT_PROG_INTLTOOL macro must be declared at toplevel so that
  `autoreconf` can detect it.

Signed-off-by: Patrick McCarty <patrick.mccarty@intel.com>
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 0434a35..cec8b7d 100644
--- a/configure.ac
+++ b/configure.ac
@@ -256,8 +256,8 @@ dnl # Disable creation and installation of translation files
 dnl #######################################################################
 AC_ARG_ENABLE(nls, AC_HELP_STRING([--disable-nls], [disable installation of translation files]), enable_i18n="$enableval", enable_i18n=yes)
 
+IT_PROG_INTLTOOL
 if test x$enable_i18n = xyes; then
-AC_PROG_INTLTOOL
 	GETTEXT_PACKAGE=pidgin
 	AC_SUBST(GETTEXT_PACKAGE)
 
